README.md

```diff
@@ -18,10 +18,29 @@ See [Hazelcast Cloud Command Line](https://github.com/hazelcast/hazelcast-cloud-
 
     $ brew tap hazelcast/hz
 
-### Install Hazelcast CLI
+### Install Hazelcast
+
+Install the community version of Hazelcast
 
     $ brew install hazelcast 
 
+Install particular version of Hazelcast
+
+    $ brew install hazelcast@5.0.1
+
+Install snapshot version of Hazelcast
+
+    $ brew install hazelcast@5.1.snapshot
+
+Install Hazelcast Enterprise edition (you need a license key to run it)
```

Comment:
What do you think about adding [link](https://docs.hazelcast.com/hazelcast/latest/getting-started/get-started-enterprise#installing-a-license-key) to the docs here: